_data/hosting_providers.json

```diff
@@ -2238,6 +2238,16 @@
     "plan": "",
     "reviewed": "2019.11.18",
     "note": ""
+  },
+    {
```

Comment:
```suggestion
  {
```

Reply:
Hi,

I guess we should go with "partial" then, because user has to click an option to generate a certificate (on cloud hosting accounts). On managed servers (hostnames) it is setup by default.

Reply:
Here's my suggestion for how to handle this situation:

- Leave it as "full"
- Add a note to the "note" section saying "Certificates provided by default for managed servers. Cloud hosting accounts offer partial support; see tutorial."
- Add a link to a tutorial or instructions for how to set up a certificate on cloud hosting accounts to the "tutorial" field.

Reply:
I do appreciate your suggestion a lot - will add that tutorial a little later, so for know it could be just left as "partial" and corrected after some time, when tutorial will be in place. Could you accept that?

Reply:
I'm fine with putting either partial or full as the category, but either way I'd appreciate putting in that note I mentioned specifying which product has which property.

Reply:
(I'd be happy to accept it as full with a tutorial to come later, as long as there's a note in there.)